packages/salesforcedx-utils-vscode/src/telemetry/interfaces/telemetryReporter.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable header/header */
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
why this copyright? if it's a MS file/type, can we import it?

Reply:
I kept this on the appInsights reporter and on the interface, since I extracted it from the original TelemetryReporter that had this copyright.